tests/src/test/java/alluxio/rest/TestCase.java

```diff
@@ -92,6 +92,13 @@ public String getMethod() {
     return mMethod;
   }
 
+  /**
+   * Create the URL which is concatenated by the mHostname, mPort, REST_API_PREFIX, mEndpoint and
```

Comment:
I think you can remove these lines (and the blank line below), since the `@return` can present the same information.